Source/WebKit/UIProcess/PageClient.h

```diff
@@ -337,6 +337,8 @@ class PageClient : public CanMakeWeakPtr<PageClient> {
     virtual WebCore::IntRect rootViewToScreen(const WebCore::IntRect&) = 0;
     virtual WebCore::IntPoint accessibilityScreenToRootView(const WebCore::IntPoint&) = 0;
     virtual WebCore::IntRect rootViewToAccessibilityScreen(const WebCore::IntRect&) = 0;
+
+    virtual void updateFindResults() = 0;
```

Comment:
This can be `virtual void updateFindResults() { }`, and then you wouldn't need to implement it everywhere.